### DISCUSSION TITLE
One menu is gone

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#10783] [Q&A]
One of the menus has been deleted, can I somehow see the logs it was deleted by one user with rights or not? 
![image](https://user-images.githubusercontent.com/12182810/144228483-10e84c55-a50c-4ec6-858f-b54c8a2b10cf.png)

Everyone who had access swears they didn't 😂

**Comment by hishamco** [comment]
I think you can catch almost if not all the events if you enable AuditTrail module

> Everyone who had access swears they didn't 😂

😂😂😂

**Reply by Piedone** [reply]
Yep, since Menus are content items. However, you'll need to enable and configure Audit Trail before the deletion, what I guess didn't quite happen here.
